platform/build-faiss/add_dev.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+sudo apt-get update && sudo apt-get dist-upgrade -y
+
+# mount nvme disk on /models
+sudo mkdir /models
+sudo mkfs.xfs /dev/nvme0n1
```

Comment:
Not a fan of destructive operations in shell scripts. Especially those that could eat the entire filesystem if in error.

Reply:
your assumption in line 30 sort of covers this. Sort of, as in no destructive operation should be done anywhere without extreme clarity.. I would prefer you pull this destructive operatoin out, as I could never run `add_dev.sh` on beast, as it would wipe my main disk...........